pkgs/servers/portunus/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "portunus";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "majewsky";
+    repo = "portunus";
+    # rev = "v${version}";
+    rev = "99dc0e3d5a84fac988759df8ba2b8cd54ffdc0fb";
+    sha256 = "sha256-CLjpiebQdEWuo5ae5KcBrzLs1hzZHiBIWOpaXdPIJAc=";
+  };
+
+  vendorSha256 = null;
+
+  postInstall = ''
+    mv $out/bin/{,portunus-}orchestrator
+    mv $out/bin/{,portunus-}server
```

Comment:
Each of these `mv` seems to take care of two binaries. Are they identical so that one can be elided?

Reply:
expanded this is ```
mv $out/bin/orchestrator $out/bin/portunus-orchestrator
mv $out/bin/server $out/bin/portunus-orchestrator
```

since I wrote this the makefile got updated and we could also use ``make build`` but that is usually not preferred for go packages.

Reply:
> Are they identical so that one can be elided?

No, Portunus explicitly has two separate binaries. One runs with root privileges, so this one is designed to pull as little libraries as possible.